sockets/1.2/modules/ROOT/pages/sockets-connector-examples.adoc

```diff
@@ -134,3 +134,125 @@ Building on the example of only sending data, in this scenario you add a new flo
 ----
 
 To test this example, use the same database setup as used for the sending example, run the Mule application, access the endpoint using the following netcat command `nc localhost 8085`, and send the data `{"id":2}`.
+
+== Create a Custom TCP Protocol
+
+In the following example, you create a Custom Protocol based on TCP that appends a string header to every message sent, and limits input message size to a fixed length, truncating all additional characters.
```

Comment:
The following example creates a TCP-based Custom Protocol that appends a ....

Should Custom Protocol be capitalized?

....and limits input message size to a fixed length by truncating additional characters.